charts/fybrik-crd/templates/app.fybrik.io_fybrikmodules.yaml

```diff
@@ -273,19 +273,32 @@ spec:
           status:
             description: FybrikModuleStatus defines the observed state of FybrikModule.
             properties:
-              errorMessage:
-                description: ErrorMessage indicates that an error has happened during
-                  the module validation
-                type: string
-              validModule:
-                description: ValidModule indicates whether the FybrikModule is valid
-                  given the defined taxonomy
-                type: string
-              validatedGeneration:
-                description: ValidatedGeneration is the version of the FybrikModule
-                  that has been validated with the taxonomy defined.
-                format: int64
-                type: integer
+              conditions:
+                description: Conditions indicate the module state after validation
+                  (Ready, Error)
+                items:
+                  description: Condition describes the state of a FybrikApplication
+                    at a certain point.
+                  properties:
+                    message:
+                      description: Message contains the details of the current condition
+                      type: string
+                    observedGeneration:
+                      description: ObservedGeneration is the version of the resource
+                        for which the condition has been evaluated
+                      format: int64
+                      type: integer
+                    status:
+                      description: 'Status of the condition: true or false'
+                      type: string
```

Comment:
If Condition status can have only 3 values: "True", "False" and "Unknown", should we add a string validation, something like: "pattern: "False|True|Unknown"

Reply:
For the best of my knowledge, we do not validate fields inside the status, only in spec.
I've used condition status similar to described in https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md
I'll change the description to include Unknown.

Reply:
Actually, there is a better option, here is an example from `issuers.cert-manager.io`
```yaml
status:
  description: Status of the condition, one of (`True`, `False`, `Unknown`).
  enum:
  - "True"
  - "False"
  - Unknown
  type: string
```

Reply:
OK